### PR TITLE
Blend ecwmf only

### PR DESCRIPTION
**Pull request by peterdudfield**
# Pull Request

## Description

If pvnet and pvnet day is not working, blend in pvnet_ecmwf only, instead of national_xg
If pvnet_ecmwf not there, use national_xg

- add the pvnet_ecmwf model, which maent updating the tests
- added pvnet_ecmwf to the weights, 
- added a test to check it works

Helps with https://github.com/openclimatefix/uk-pvnet-app/issues/123

## How Has This Been Tested?

CI tests + added extra

- [x] Yes

## Checklist:

- [x] My code follows [OCF's coding style guidelines](https://github.com/openclimatefix/.github/blob/main/coding_style.md)
- [x] I have performed a self-review of my own code
- [ ] I have made corresponding changes to the documentation
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] I have checked my code and corrected any misspellings

### CODE REVIEW

File: forecast_blend/weights.py [hunk 1]
```diff
@@ -35,24 +17,50 @@
...
         # pvnet_day_ahead
         "start_horizon_hour": 8,
         "end_horizon_hour": 36,
-        "start_weight": [0, 1, 0],
-        "end_weight": [0, 1, 0],
+        "start_weight": [0, 1, 0, 0],
+        "end_weight": [0, 1, 0, 0],
+    },
+    {
+        # backup to use ecmwf
```
Comment (peterdudfield) [inline forecast_blend/weights.py:37]: add more details here

File: readme.md [hunk 2]
```diff
@@ -18,6 +18,7 @@ but we only update the ForecastValue table every 30 minutes
 - Note we only blend forecasts if they are made within 6 hours. 
 If all forecasts are older than this, then all forecasts are used.
 - The probabilistic forecasts are now blended using the same method as the expected value
+- We current intial take `Pvnet`, then blend it with `PVnet DA`. `PVnet ECMWF` is used as a back up, and final `National-xg` is a final backup.
```
Comment (Sukh-P) [inline readme.md:21]: Small typo here

**Review by Sukh-P — APPROVED** [review]
Nice work adding this in here!

File: tests/conftest.py [hunk 3]
```diff
@@ -57,6 +57,33 @@ def forecast_national(db_session):
...
 
+@pytest.fixture
+def forecast_national_ecmwf_and_xg(db_session):
+    t0_datetime_utc = (datetime.now(tz=timezone.utc) + timedelta(days=2)).replace(minute=0, second=0, microsecond=0)
+    # time detal of 2 days is used as fake forecast are made 2 days in the past,
+    # this makes them for now
+    # create
+    model_names_ecmwf_and_xg = ["pvnet_ecmwf", "National_xg"]
+    for i in range(len(model_names_ecmwf_and_xg)):
+        model_name = model_names_ecmwf_and_xg[i]
```
Comment (Sukh-P) [inline tests/conftest.py:68]: Could simplify this to
```suggestion
    for model_name in model_names_ecmwf_and_xg:
```

**Pull request merged**